tests/MakersOfDenmark.WebAPI.Tests/SearchForMakerSpaceHandlerTest.cs

```diff
@@ -40,8 +40,8 @@ public async Task SearchForOneMakerSpaceTest()
         public async Task SearchForManyMakerSpacesTest()
         {
             //Arrange
-            var makerSpaceOne = _requestHandlerFixture.Fixture.Build<MakerSpace>().With(x => x.Name == "Aarhus Universitet").With(x => x.Address, new Address("Test Street", "Test City", "Test Country", "Test Postcode")).Without(x => x.ContactInfo).Without(x => x.VATNumber).Without(x => x.Organization).Without(x => x.Tools).Create();
-            var makerSpaceTwo = _requestHandlerFixture.Fixture.Build<MakerSpace>().With(x => x.Name == "Aalborg Universitet").With(x => x.Address, new Address("Test Street", "Test City", "Test Country", "Test Postcode")).Without(x => x.ContactInfo).Without(x => x.VATNumber).Without(x => x.Organization).Without(x => x.Tools).Create();
+            var makerSpaceOne = _requestHandlerFixture.Fixture.Build<MakerSpace>().With(x => x.Name, () => "Aarhus Universitet").With(x => x.Address, new Address("Test Street", "Test City", "Test Country", "Test Postcode")).Without(x => x.ContactInfo).Without(x => x.VATNumber).Without(x => x.Organization).Without(x => x.Tools).Create();
+            var makerSpaceTwo = _requestHandlerFixture.Fixture.Build<MakerSpace>().With(x => x.Name, () => "Aalborg Universitet").With(x => x.Address, new Address("Test Street", "Test City", "Test Country", "Test Postcode")).Without(x => x.ContactInfo).Without(x => x.VATNumber).Without(x => x.Organization).Without(x => x.Tools).Create();
```

Comment:
syntaks på .With() er:
`(kode kommer forud).With(x=>x.Name, "Tekst-streng som navn").(kode der kommer bagefter)`

Reply:
Det er bare et forslag til en mere kompakt syntaks

Reply:
Da adresse ikke er vigtig her, kan du også lave den med Fixture og smide den ind (i begge makerspaces

Reply:
Du burde måske have et 3. makerspace med et helt andet navn.
- Så du har tre i listen, men den skal kun finde to.
- Det er bare lige et lille ekstre tjek på at funktionaliteten er iorden. Hvis du får ingen eller 3 så er der jo noget galt :)
(testen er dog iorden)

Reply:
Rigtig god idé med et tredje makerspace som ekstra validering! Samt gode forslag til mere kompakt kode.

Reply:
<img width="642" alt="address" src="https://user-images.githubusercontent.com/32497085/97166619-d6658a00-1785-11eb-8d11-17a7272000ec.png">
Udfordring med at lave én adresse som sættes ind ved begge fixtures. Hvis nogen har et forslag/idé eller kan se hvad jeg gør galt, hører jeg gerne

Reply:
prøv:
```
var address = _requestHandlerFixture.Fixture.Create<Address>();
```
Fixture opretter din addresse - det er ikke vigtigt hvad der er inde i den :)

I princippet brude Fixture kunne oprette en MakerSpace incl address, så du helt slipper for den ekstra linje, du er jo kun interesseret i at styre dens Name
```
var makerSpaceOne = ...Fixture.Build<MakerSpace>().With(x=>x.Name, "Gøgler university").Create();
```

Reply:
For du kan gøre ovenstående, skal du muligvis bruge disse 3 linjer i toppen af din test: 
```
_requestHandlerFixture.Fixture.Behaviors.OfType<ThrowingRecursionBehavior>().ToList().ForEach(b 
    =>_requestHandlerFixture.Fixture.Behaviors.Remove(b));
_requestHandlerFixture.Fixture.Behaviors.Add(new OmitOnRecursionBehavior());
```

Reply:
Tusind tak for hjælpen @jesp209i det løste det.